pkg/apis/application/v1alpha1/types.go

```diff
@@ -1432,6 +1432,16 @@ type SyncPolicy struct {
 	// If you add a field here, be sure to update IsZero.
 }
 
+// IsAutomatedSyncEnabled checks if the automated sync is enabled or disabled
+func (p *SyncPolicy) IsAutomatedSyncEnabled() bool {
+	var isEnabled bool
+
+	if p.Automated != nil && (p.Automated.Enable == nil || *p.Automated.Enable) {
+		isEnabled = true
```

Comment:
better to `return true` and remove the variable `isEnabled`

Reply:
Updated 👍🏽